examples/angular/src/app/app.component.ts

```diff
@@ -87,7 +63,17 @@ export class AppComponent implements OnInit {
     const _modal = setupModal(_selector, { contractId: CONTRACT_ID });
     const state = _selector.store.getState();
 
-    this.syncAccountState(localStorage.getItem("accountId"), state.accounts);
+    if (!state.accounts.length) {
+      this.accountId = null;
+      this.accounts = [];
+    }
+
+    const newAccount = state.accounts.find((account) => account.active);
+
+    if (newAccount) {
+      this.accountId = newAccount.accountId;
+      this.accounts = state.accounts;
+    }
```

Comment:
Regarding the comment, in Angular, this will be a problem:
```ts
if (prevAccountId !== this.accountId) {
  this.getAccount().then((account) => {
    this.account = account;
  });
}
```